### PR TITLE
Add VS Code debugging support to metaphysics + docs

### PR DESCRIPTION
**Pull request by orta**
Will comment what each file does

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"christian-kohler.path-intellisense",
+		"christian-kohler.npm-intellisense",
+		"dbaeumer.vscode-eslint",
+		"flowtype.flow-for-vscode"
+	]
+}
```
Comment (orta) [inline .vscode/extensions.json]: JS-y recommendations for extensions, Flow is used for internal typings but isn't "active" on the project, will just offer some useful autocompletions via the default type system

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            "port": 5858,
+            "address": "localhost",
+            "restart": true,
+            "sourceMaps": false,
+            "outFiles": [],
+            "localRoot": "${workspaceRoot}",
+            "remoteRoot": null
+        }
+    ]
+}
```
Comment (orta) [inline .vscode/launch.json]: As we run with node foreman, and the nodemon - we can't easily run it as a launchable process in VS Code, however we can successfully connect and reconnect to a running node process, which works fine 👍

File: .vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+            "taskName": "Start Node Foreman in Dev",
+            "args": [
+                "run",
+                "dev"
+            ],
+            "isWatching": true,
+            "isBuildCommand": true
+        }
+    ]
+}
```
Comment (orta) [inline .vscode/tasks.json]: This _lets_ you run the project through VS Code if you choose - it is optional, just runs `npm run dev` and keeps results inline.

File: README.md [hunk 4]
```diff
@@ -4,15 +4,15 @@ Metaphysics is a [GraphQL](http://graphql.org)-compliant API for the Artsy v1 AP
...
+It is currently used in production all over the place in [Artsy.net](http://github.com/artsy/force/), and the [Artsy iOS App](http://github.com/artsy/eigen)
 
 ### Meta
 
 * __State:__ production
 * __Production:__ [metaphysics-production.artsy.net](https://metaphysics-production.artsy.net/)
 * __Staging:__ [metaphysics-staging.artsy.net](https://metaphysics-staging.artsy.net/)
 * __CI:__ [Semaphore](https://semaphoreapp.com/artsy-it/metaphysics/)
-* __Point People:__ [@dzucconi](https://github.com/dzucconi) & [@broskoski](https://github.com/broskoski)
+* __Point People:__ [@alloy](https://github.com/alloy) & [@mzikherman](https://github.com/mzikherman)
```
Comment (orta) [inline README.md]: You two seem to be the most reliable sources of authority

Comment (alloy) [inline README.md]: We’re definitely around! 😜

**Comment by alloy** [comment]
Awesome 👌

**Pull request merged**